### ISSUE TITLE
Timezone Issue

### ISSUE THREAD

**Issue by chrisalbon** [picocms/Pico#68]
Since installing Pico, I get this issue on the top of every page:

```
Warning: strtotime() [function.strtotime]: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/New_York' for 'EDT/-4.0/DST' instead in /hsphere/local/home/livenoma/writemorefaster.com/lib/pico.php on line 166

Warning: date() [function.date]: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/New_York' for 'EDT/-4.0/DST' instead in /hsphere/local/home/livenoma/writemorefaster.com/lib/pico.php on line 166

Warning: strtotime() [function.strtotime]: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/New_York' for 'EDT/-4.0/DST' instead in /hsphere/local/home/livenoma/writemorefaster.com/lib/pico.php on line 166

Warning: date() [function.date]: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/New_York' for 'EDT/-4.0/DST' instead in /hsphere/local/home/livenoma/writemorefaster.com/lib/pico.php on line 166

Warning: strtotime() [function.strtotime]: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/New_York' for 'EDT/-4.0/DST' instead in /hsphere/local/home/livenoma/writemorefaster.com/lib/pico.php on line 234

Warning: date() [function.date]: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/New_York' for 'EDT/-4.0/DST' instead in /hsphere/local/home/livenoma/writemorefaster.com/lib/pico.php on line 234

Warning: strtotime() [function.strtotime]: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/New_York' for 'EDT/-4.0/DST' instead in /hsphere/local/home/livenoma/writemorefaster.com/lib/pico.php on line 234


Warning: date() [function.date]: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/New_York' for 'EDT/-4.0/DST' instead in /hsphere/local/home/livenoma/writemorefaster.com/lib/pico.php on line 234

Warning: strtotime() [function.strtotime]: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/New_York' for 'EDT/-4.0/DST' instead in /hsphere/local/home/livenoma/writemorefaster.com/lib/pico.php on line 234

Warning: date() [function.date]: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected 'America/New_York' for 'EDT/-4.0/DST' instead in /hsphere/local/home/livenoma/writemorefaster.com/lib/pico.php on line 234
```

**Comment by dav-m85** [comment]
@theshka well I find a bit harsh to close all issues without explaining quickly why to the community. :-1:

**Comment by mbp2011** [comment]
Many of these threads are 2 years old with no new activity. In the interest of cleaning up I quickly reviewed the threads and closed any that I thought were resolved or abandoned. I will gladly reopen any issue or pull request anyone raises issue with. 

Regarding this thread specifically, the `config.php.template` file now contains `date_default_timezone_set('UTC');` and I figured this issue was resolved. 

Thanks, I hope this clears up any confusion @dav-m85

**Comment by theshka** [comment]
Many of these threads are 2 years old with no new activity. In the interest of cleaning up I quickly reviewed the threads and closed any that I thought were resolved or abandoned. I will gladly reopen any issue or pull request anyone raises issue with. 

Regarding this thread specifically, the `config.php.template` file now contains `date_default_timezone_set('UTC');` and I figured this issue was resolved. 

Thanks, I hope this clears up any confusion @dav-m85
